2025.csv

```diff
@@ -35,3 +35,4 @@ PyCon Estonia,2025-10-02,2025-10-03,"Tallinn, Estonia",EST,,,2025-01-15,https://
 Swiss Python Summit,2025-10-16,2025-10-17,"Rapperswil, Switzerland",CHE,OST Campus,,2025-05-31,https://www.python-summit.ch,https://www.python-summit.ch/cfp/,https://www.python-summit.ch/sponsoring/
 Python Brasil,2025-10-21,2025-10-27,"Sao Paulo, Brazil",BRA,São Luís Events Center,2025-04-27,2025-04-27,https://2025.pythonbrasil.org.br/,https://talks.python.org.br/pythonbrasil-2025/cfp,https://www.canva.com/design/DAGe_yZ0SdI/myPIGUAEQAnrwKhJE_7Bag/view?utm_content=DAGe_yZ0SdI&utm_campaign=designshare&utm_medium=link2&utm_source=uniquelinks&utlId=h7f3e8c2330
 PyCon Sweden,2025-10-30,2025-10-31,"Stockholm, Sweden",SWE,Clarion Skanstull Hotel Stockholm,,,https://pycon.se/,,https://pycon.se/#sponsorship
+EuroSciPy,2025-08-18,2024-08-22,"Krakow, Poland",POL,AGH University,2025-04-28,2025-04-28,https://www.euroscipy.org/2025/,https://www.euroscipy.org/2025/program.html,https://www.euroscipy.org/2025/sponsoring.html
```

Comment:
Please move this line to be between DjangoCon Africa and PyCon Greece. Thanks.

Reply:
Ah right, done!